templates/vue/src/components/tyde/Navbar.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="nav-container">
+    <b-navbar tyde="light" class="navbar">
+      <b-navbar-nav class="mx-auto" fill style="width:100%;">
+        <b-nav-item
+          v-for="tab in tabs"
+          :key="tab.name"
+          :ref="tab.name + '-tab'"
+          :to="tab.link"
+          active-class="active"
+          link-classes="link"
+        >
+          <tyde-icon class="selected" :icon="tab.name" />
+          <tyde-icon class="unselected" :icon="tab.name + '-unselected'" />
```

Comment:
The reason this is being done with css is because active is only set as a class and is not accessible by JS code. 
I am looking for an idea to improve this to use v-if instead, open for suggestions.